test/packages/with-kind/kubernetes/data_stream/state_cronjob/fields/agent.yml

```diff
@@ -0,0 +1,198 @@
+- name: cloud
```

Comment:
Since it's only for testing, could we reduce the amount of fields we have in this file?

Reply:
We can, but it was left as it is to be in sync with Kubernetes integration. Should I update it?